TramsDataApi/DatabaseModels/A2BSchoolLoan.cs

```diff
@@ -7,11 +7,13 @@ namespace TramsDataApi.DatabaseModels
     public class A2BSchoolLoan
     {
         [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
```

Comment:
this should be a int, I'm not sure how ef core/ ms sql would create an identity string column, but I would imagine that it could cause issues...